apps/game_backend/lib/game_backend/matches.ex

```diff
@@ -114,6 +100,40 @@ defmodule GameBackend.Matches do
     end)
   end
 
+  defp complete_or_fail_bounties(multi, results) do
```

Comment:
Shouldn't this go under
```
  ####################
  #      Helpers     #
  ####################
```
?

Reply:
Same goes for the previous function, even though you didn't create it you did modify it so you might as well move it to the right place

Reply:
There are under the # Multi operations # title since they affect the match insertion multi, they are not helpers